wondrous-app/services/otterspace/index.ts

```diff
@@ -0,0 +1,99 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+
+import { useWonderWeb3 } from '../web3';
+
+const GUILD_BASE_URL = 'https://api.thegraph.com/subgraphs/name/otterspace-xyz/badges-goerli';
+
+const useOtterspace = () => {
```

Comment:
This is how I did the CyberConnect stuff, but actually you can reference how we did the Snapshot client graphql queries - pretty neat.